mutiny-core/src/chain.rs

```diff
@@ -52,6 +52,7 @@ impl<S: MutinyStorage> BroadcasterInterface for MutinyChain<S> {
         let logger = self.logger.clone();
         utils::spawn(async move {
             for tx in txs_clone {
+                log_info!(logger, "Broadcasting transaction: {}", tx.txid());
```

Comment:
actually tx hex would be useful too, to see what it is trying to spent in case it has inputs that can't be spent yet, or at all.

Reply:
added